Jenkinsfile-bleeding-edge-monthly

```diff
@@ -512,15 +469,15 @@ pipeline {
         }
 
     }
-
-    post {
-        always {
-            node("linux") {
-                recordIssues enabledForFailure: false, tool: clang()
-            }
-        }
-        success { script { utils.mailBuildResults("SUCCESSFUL") } }
-        unstable { script { utils.mailBuildResults("UNSTABLE") } }
-        failure { script { utils.mailBuildResults("FAILURE") } }
-    }
+    // post {

```

Comment:
We should re-enable this.

Could we also have a set list of people get emailed from the monthly run?

Reply:
Uncommenting this part now.
We can pass a custom list of additional emails: https://github.com/stan-dev/jenkins-shared-libraries/blob/master/src/org/stan/Utils.groovy#L236